packages/core/database/lib/entity-manager.js

```diff
@@ -108,38 +108,43 @@ const processData = (metadata, data = {}, { withDefaults = false } = {}) => {
   return obj;
 };
 
-const createEntityManager = db => {
+const createEntityManager = (db, trx) => {
```

Comment:
The `createEntityManager` should take the `lock` parameter and pass it to the `createQueryBuilder`. 
And the `queryBuilder` should pass it along the `trx` everywhere and into the `getConnection` method call.